README.md
# NAME

Task::MapMetro::Maps - Map::Metro maps in a bundle

<div>
    <p>
    <img src="https://img.shields.io/badge/perl-5.10+-blue.svg" alt="Requires Perl 5.10+" />
    <a href="https://travis-ci.org/Csson/p5-Task-MapMetro-Maps"><img src="https://api.travis-ci.org/Csson/p5-Task-MapMetro-Maps.svg?branch=master" alt="Travis status" /></a>
    <a href="http://cpants.cpanauthors.org/dist/Task-MapMetro-Maps-0.1302"><img src="https://badgedepot.code301.com/badge/kwalitee/Task-MapMetro-Maps/0.1302" alt="Distribution kwalitee" /></a>
    <a href="http://matrix.cpantesters.org/?dist=Task-MapMetro-Maps%200.1302"><img src="https://badgedepot.code301.com/badge/cpantesters/Task-MapMetro-Maps/0.1302" alt="CPAN Testers result" /></a>
    </p>
</div>

# VERSION

Version 0.1302, released 2016-05-14.

# TASK CONTENTS

## Map::Metro

### [Map::Metro](https://metacpan.org/pod/Map::Metro) 0.2404

## Available maps

### [Map::Metro::Plugin::Map::Amsterdam](https://metacpan.org/pod/Map::Metro::Plugin::Map::Amsterdam)

### [Map::Metro::Plugin::Map::Athens](https://metacpan.org/pod/Map::Metro::Plugin::Map::Athens)

### [Map::Metro::Plugin::Map::Barcelona](https://metacpan.org/pod/Map::Metro::Plugin::Map::Barcelona)

### [Map::Metro::Plugin::Map::Berlin](https://metacpan.org/pod/Map::Metro::Plugin::Map::Berlin)

### [Map::Metro::Plugin::Map::Brussels](https://metacpan.org/pod/Map::Metro::Plugin::Map::Brussels)

### [Map::Metro::Plugin::Map::Budapest](https://metacpan.org/pod/Map::Metro::Plugin::Map::Budapest)

### [Map::Metro::Plugin::Map::Copenhagen](https://metacpan.org/pod/Map::Metro::Plugin::Map::Copenhagen)

### [Map::Metro::Plugin::Map::Gothenburg](https://metacpan.org/pod/Map::Metro::Plugin::Map::Gothenburg)

### [Map::Metro::Plugin::Map::Helsinki](https://metacpan.org/pod/Map::Metro::Plugin::Map::Helsinki)

### [Map::Metro::Plugin::Map::Lille](https://metacpan.org/pod/Map::Metro::Plugin::Map::Lille)

### [Map::Metro::Plugin::Map::Lyon](https://metacpan.org/pod/Map::Metro::Plugin::Map::Lyon)

### [Map::Metro::Plugin::Map::Madrid](https://metacpan.org/pod/Map::Metro::Plugin::Map::Madrid)

### [Map::Metro::Plugin::Map::Oslo](https://metacpan.org/pod/Map::Metro::Plugin::Map::Oslo)

### [Map::Metro::Plugin::Map::Stockholm](https://metacpan.org/pod/Map::Metro::Plugin::Map::Stockholm)

# SYNOPSIS

    $ cpanm Task::MapMetro::Maps

# SEE ALSO

[Map::Metro](https://metacpan.org/pod/Map::Metro)

[Task::MapMetro::Dev](https://metacpan.org/pod/Task::MapMetro::Dev)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT

Copyright 2015 - Erik Carlsson

# LICENSE

This library is free software; you can redistribute it and/or modify
it under the same terms as Perl itself.

# SOURCE

[https://github.com/Csson/p5-Task-MapMetro-Maps](https://github.com/Csson/p5-Task-MapMetro-Maps)

# HOMEPAGE

[https://metacpan.org/release/Task-MapMetro-Maps](https://metacpan.org/release/Task-MapMetro-Maps)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2016 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Task-MapMetro-Maps

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.1302 2016-05-14T21:35:23Z UTC
 [Contents]
 - Temporarily removed Bilbao and Brescia (currently not available).
 - Bumped Map::Metro to 0.2404.

 [Stats & Dependency Changes]
 - 4 files changed, 58 insertions(+), 24 deletions(-)
 - (run req) ~ Map::Metro 0.2206 --> 0.2404
 - (run req) ~ perl 5.014000 --> 5.010000
 - (run req) - Map::Metro::Plugin::Map::Bilbao
 - (run req) - Map::Metro::Plugin::Map::Brescia
 - (test req) + ExtUtils::MakeMaker (any)
 - (test req) + File::Spec (any)
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (test rec) + CPAN::Meta 2.120900
 - (conf req) + ExtUtils::MakeMaker (any)
 - (conf req) - Module::Build::Tiny
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Cpantesters (any)
 - (dev req) + Badge::Depot::Plugin::Kwalitee (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Badge::Depot::Plugin::Travis (any)
 - (dev req) + Dist::Zilla 5.000
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.007
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0100
 - (dev req) + Dist::Zilla::Plugin::CheckChangesHasContent 0.008
 - (dev req) + Dist::Zilla::Plugin::Clean 0.07
 - (dev req) + Dist::Zilla::Plugin::ConfirmRelease (any)
 - (dev req) + Dist::Zilla::Plugin::CopyFilesFromBuild 0.151680
 - (dev req) + Dist::Zilla::Plugin::DistIller::MetaGeneratedBy (any)
 - (dev req) + Dist::Zilla::Plugin::ExecDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git 2.036
 - (dev req) + Dist::Zilla::Plugin::Git::Check (any)
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.013
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Contributors 0.017
 - (dev req) + Dist::Zilla::Plugin::Git::GatherDir (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Push (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Tag (any)
 - (dev req) + Dist::Zilla::Plugin::GithubMeta 0.54
 - (dev req) + Dist::Zilla::Plugin::InstallRelease 0.008
 - (dev req) + Dist::Zilla::Plugin::License (any)
 - (dev req) + Dist::Zilla::Plugin::MakeMaker (any)
 - (dev req) + Dist::Zilla::Plugin::Manifest (any)
 - (dev req) + Dist::Zilla::Plugin::ManifestSkip (any)
 - (dev req) + Dist::Zilla::Plugin::MetaConfig (any)
 - (dev req) + Dist::Zilla::Plugin::MetaJSON (any)
 - (dev req) + Dist::Zilla::Plugin::MetaNoIndex (any)
 - (dev req) + Dist::Zilla::Plugin::MetaProvides::Package 2.003001
 - (dev req) + Dist::Zilla::Plugin::MetaYAML (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::NextVersion::Semantic (any)
 - (dev req) + Dist::Zilla::Plugin::OurPkgVersion (any)
 - (dev req) + Dist::Zilla::Plugin::PodSyntaxTests (any)
 - (dev req) + Dist::Zilla::Plugin::Prereqs (any)
 - (dev req) + Dist::Zilla::Plugin::PreviousVersion::Changelog (any)
 - (dev req) + Dist::Zilla::Plugin::Readme (any)
 - (dev req) + Dist::Zilla::Plugin::ReadmeAnyFromPod 0.150250
 - (dev req) + Dist::Zilla::Plugin::ReversionOnRelease (any)
 - (dev req) + Dist::Zilla::Plugin::RunExtraTests 0.028
 - (dev req) + Dist::Zilla::Plugin::ShareDir (any)
 - (dev req) + Dist::Zilla::Plugin::TaskWeaver 0.101628
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.054
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::ReportPrereqs 0.021
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.05
 - (dev req) + Dist::Zilla::Plugin::TestRelease (any)
 - (dev req) + Dist::Zilla::Plugin::UploadToCPAN (any)
 - (dev req) + Pod::Elemental::Transformer::List (any)
 - (dev req) + Pod::Weaver::Plugin::SingleEncoding (any)
 - (dev req) + Pod::Weaver::Plugin::Transformer (any)
 - (dev req) + Pod::Weaver::PluginBundle::CorePrep (any)
 - (dev req) + Pod::Weaver::Section::Authors (any)
 - (dev req) + Pod::Weaver::Section::Badges 0.0401
 - (dev req) + Pod::Weaver::Section::Collect (any)
 - (dev req) + Pod::Weaver::Section::GenerateSection 1.01
 - (dev req) + Pod::Weaver::Section::Generic (any)
 - (dev req) + Pod::Weaver::Section::Leftovers (any)
 - (dev req) + Pod::Weaver::Section::Legal (any)
 - (dev req) + Pod::Weaver::Section::Name (any)
 - (dev req) + Pod::Weaver::Section::Region (any)
 - (dev req) + Pod::Weaver::Section::Version (any)
 - (dev req) + Test::EOF (any)
 - (dev req) + Test::EOL (any)
 - (dev req) + Test::More 0.88
 - (dev req) + Test::NoTabs 1.4
 - (dev req) + Test::Version 1
 - (dev req) + Test::Warnings 0.026
 - (dev sug) + Dist::Iller 0.1408
 - (dev sug) + Dist::Iller::Config::Author::CSSON 0.0309

0.1301 2015-02-03T22:30:36Z CET
 [Enhancements]
 - Added Lyon.

 [Requirements]
 - Bumped Map::Metro to 0.2206.

0.1300 2015-01-09T23:36:44Z CET
 [Enhancements]
 - Bumped Map::Metro to 0.2201.
 - Added Lille.

0.1200 2015-01-08T21:53:15Z CET
 [New Features]
 - Added Budapest and Madrid.

0.1100 2015-01-07T01:40:32Z CET
 [New Features]
 - Added Bilbao, Brescia and Brussels.

0.1000 2015-01-05T16:13:44Z CET
 - Initial release


lib/Task/MapMetro/Maps.pm
use strict;
use warnings;

package Task::MapMetro::Maps;

# ABSTRACT: Map::Metro maps in a bundle
# AUTHORITY
our $VERSION = '0.1303';

1;


__END__

=encoding utf-8

=head1 SYNOPSIS

    $ cpanm Task::MapMetro::Maps

=pkgroup Map::Metro

=pkg Map::Metro 0.2404

=pkgroup Available maps

=pkg Map::Metro::Plugin::Map::Amsterdam

=pkg Map::Metro::Plugin::Map::Athens

=pkg Map::Metro::Plugin::Map::Barcelona

=pkg Map::Metro::Plugin::Map::Berlin

=pkg Map::Metro::Plugin::Map::Brussels

=pkg Map::Metro::Plugin::Map::Budapest

=pkg Map::Metro::Plugin::Map::Copenhagen

=pkg Map::Metro::Plugin::Map::Gothenburg

=pkg Map::Metro::Plugin::Map::Helsinki

=pkg Map::Metro::Plugin::Map::Lille

=pkg Map::Metro::Plugin::Map::Lyon

=pkg Map::Metro::Plugin::Map::Madrid

=pkg Map::Metro::Plugin::Map::Oslo

=pkg Map::Metro::Plugin::Map::Stockholm

=head1 SEE ALSO

L<Map::Metro>

L<Task::MapMetro::Dev>

=head1 AUTHOR

Erik Carlsson E<lt>info@code301.comE<gt>

=head1 COPYRIGHT

Copyright 2015 - Erik Carlsson

=head1 LICENSE

This library is free software; you can redistribute it and/or modify
it under the same terms as Perl itself.

=cut


Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.044.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "Map::Metro maps in a bundle",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Task-MapMetro-Maps",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Task::MapMetro::Maps",
  "PREREQ_PM" => {
    "Map::Metro" => "0.2404",
    "Map::Metro::Plugin::Map::Amsterdam" => 0,
    "Map::Metro::Plugin::Map::Athens" => 0,
    "Map::Metro::Plugin::Map::Barcelona" => 0,
    "Map::Metro::Plugin::Map::Berlin" => 0,
    "Map::Metro::Plugin::Map::Brussels" => 0,
    "Map::Metro::Plugin::Map::Budapest" => 0,
    "Map::Metro::Plugin::Map::Copenhagen" => 0,
    "Map::Metro::Plugin::Map::Gothenburg" => 0,
    "Map::Metro::Plugin::Map::Helsinki" => 0,
    "Map::Metro::Plugin::Map::Lille" => 0,
    "Map::Metro::Plugin::Map::Lyon" => 0,
    "Map::Metro::Plugin::Map::Madrid" => 0,
    "Map::Metro::Plugin::Map::Oslo" => 0,
    "Map::Metro::Plugin::Map::Stockholm" => 0
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.1303",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Map::Metro" => "0.2404",
  "Map::Metro::Plugin::Map::Amsterdam" => 0,
  "Map::Metro::Plugin::Map::Athens" => 0,
  "Map::Metro::Plugin::Map::Barcelona" => 0,
  "Map::Metro::Plugin::Map::Berlin" => 0,
  "Map::Metro::Plugin::Map::Brussels" => 0,
  "Map::Metro::Plugin::Map::Budapest" => 0,
  "Map::Metro::Plugin::Map::Copenhagen" => 0,
  "Map::Metro::Plugin::Map::Gothenburg" => 0,
  "Map::Metro::Plugin::Map::Helsinki" => 0,
  "Map::Metro::Plugin::Map::Lille" => 0,
  "Map::Metro::Plugin::Map::Lyon" => 0,
  "Map::Metro::Plugin::Map::Madrid" => 0,
  "Map::Metro::Plugin::Map::Oslo" => 0,
  "Map::Metro::Plugin::Map::Stockholm" => 0,
  "Test::More" => "0.96"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);
